services/cosi/cosi.go

```diff
@@ -48,13 +48,13 @@ type SignatureResponse struct {
 // SignatureRequest treats external request to this service.
 func (cs *CoSi) SignatureRequest(si *network.ServerIdentity, req *SignatureRequest) (network.Body, error) {
 	tree := req.Roster.GenerateBinaryTree()
-	tni := cs.NewTreeNodeInstance(tree, tree.Root, protocol.Name)
-	pi, err := protocol.NewCoSi(tni)
+	tni := cs.NewTreeNodeInstance(tree, tree.Root, cosi.Name)
+	pi, err := cosi.NewCoSi(tni)
```

Comment:
Perhaps now that it's called `cosi` we should rename `NewCosi`, so it's more like `cosi.NewProtocol`?